specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2020-11-10-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,4433 @@
+            "DepthFirst"
+          ],
+          "x-ms-enum": {
+            "name": "SessionHostLoadBalancingAlgorithm",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Load balancing algorithm for peak period."
+        },
+        "rampDownStartTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Starting time for ramp down period."
+        },
+        "rampDownLoadBalancingAlgorithm": {
+          "enum": [
+            "BreadthFirst",
+            "DepthFirst"
+          ],
+          "x-ms-enum": {
+            "name": "SessionHostLoadBalancingAlgorithm",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Load balancing algorithm for ramp down period."
+        },
+        "rampDownMinimumHostsPct": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 100,
+          "description": "Minimum host percentage for ramp down period."
+        },
+        "rampDownCapacityThresholdPct": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 100,
+          "description": "Capacity threshold for ramp down period."
+        },
+        "rampDownForceLogoffUsers": {
+          "type": "boolean",
+          "description": "Should users be logged off forcefully from hosts."
+        },
+        "rampDownStopHostsWhen": {
+          "enum": [
+            "ZeroSessions",
+            "ZeroActiveSessions"
+          ],
+          "x-ms-enum": {
+            "name": "StopHostsWhen",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Specifies when to stop hosts during ramp down period."
+        },
+        "rampDownNotificationMinutes": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of minutes to wait to stop hosts during ramp down period."
+        },
+        "rampDownNotificationMessage": {
+          "type": "string",
+          "description": "Notification message for users during ramp down period."
+        },
+        "offPeakStartTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Starting time for off-peak period."
+        },
+        "offPeakLoadBalancingAlgorithm": {
+          "enum": [
+            "BreadthFirst",
+            "DepthFirst"
+          ],
+          "x-ms-enum": {
+            "name": "SessionHostLoadBalancingAlgorithm",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Load balancing algorithm for off-peak period."
+        }
+      }
+    },
+    "ScalingHostPoolReference": {
+      "description": "Scaling plan reference to hostpool.",
+      "type": "object",
+      "properties": {
+        "hostPoolArmPath": {
+          "type": "string",
```

Comment:
please ensure there is a linked access check on this to validate that the person creating the scaling plan has appropriate permissions on the host pools it is being applied to